src/net/constants.rs

```diff
@@ -11,4 +10,10 @@ pub const FRAGMENT_SIZE_DEFAULT: u16 = 1024;
 ///
 /// This is not strictly guaranteed -- there may be less room in an ethernet frame than this due to
 /// variability in ipv6 header size.
-pub const DEFAULT_MTU: u16 = 1452;
\ No newline at end of file
+pub const DEFAULT_MTU: u16 = 1452;
+/// This is the current protocol version.
+///
+/// It is used for:
+/// - Generating crc32 for the packet header.
+/// - Validating if arriving packets have the same protocol version.
+pub const PROTOCOL_VERSION: &'static str = "laminar-0.1.0";
```

Comment:
I want to think if there is a better way to do this. 🤔 . This is fine for now but if we do merge this, lets open an issue to follow up on.

Reply:
What is it that you don't like about it. You want somehow identify the current protocol I have seen this in a couple of libraries done like this. But if it is an issue lets discuss it in an issue.

Reply:
Its hard coded and I'd prefer to find a better place maybe to write this.

Reply:
I think the idea would be that it gets bumped here if the protocol changes? I'm not sure where else you could put it, unless you wanted to break it up into multiple variables and assemble a String, but not sure how that would help.

Reply:
We could use a file called VERSION that is at the root and use `include_bytes!()`